test/core/authorship/block_builder_test.cpp

```diff
@@ -44,17 +45,19 @@ class BlockBuilderTest : public ::testing::Test {
  * @given BlockBuilderApi that fails to apply extrinsic @and BlockBuilder that
  * uses that BlockBuilderApi
  * @when BlockBuilder tries to push extrinsic @and BlockBuilder bakes a block
- * @then push fails
+ * @then push fails @and created block is empty
```

Comment:
```suggestion
 * @then push fails, and created block is empty
```

Reply:
@and works as well. These approach is used in Iroha and I just follow it

Reply:
ok, I didn't know